src/settings/ChangePassword/ChangePassword.test.js

```diff
@@ -9,7 +9,7 @@ import {
   fireEvent,
 } from '@folio/jest-config-stripes/testing-library/react';
 
-import { stripesConnect } from '@folio/stripes/core';
+// import { stripesConnect } from '@folio/stripes/core';
```

Comment:
Petty: I know it's just a test, but please omit code rather than commenting it unless there is a specific reason to keep it around.

Reply:
Thank you.. My bad.. I have removed that line.